projects/fsastorefrontlib/src/assets/translations/en/b2b.en.ts

```diff
@@ -148,6 +151,115 @@ export const orgUnit = {
   },
 };
 
+export const orgCostCenter = {
+  orgCostCenter: {
+    title: 'Asssigned Products',
+    subtitle: 'Products already assigned to the unit.',
+    header: 'All cost centers ({{count}})',
+    activate: 'Assign',
+    deassign: 'Deassign',
+
+    code: 'Code',
+    active: 'Status',
+    name: 'Name',
+    currency: 'Currency',
+    unit: 'Parent Unit',
+    actions: '',
+
+    byName: 'Sort by name',
+    byCode: 'Sort by code',
+    byUnitName: 'Sort by unit',
+
+    disable: {
+      confirm: 'Disable',
+    },
+
+    messages: {
+      deactivate: 'Are you sure you want to disable this cost center?',
```

Comment:
We dont need messes regarding cost centers and budgets.

Reply:
You are right that we don't need all of these messages, but the ones regarding title, subtitle, header etc, we do. This is coupled with the enum that is defined for cost_center that we used for our purpose, but cannot rename or override.